src/mapping/findpoint.jl

```diff
@@ -0,0 +1,54 @@
+abstract type AbstractPointFinderStrategy end
+struct StrictPointFinderStrategy <: AbstractPointFinderStrategy end
+struct ClosestPointFinderStrategy <: AbstractPointFinderStrategy end
+
+struct PointFinder{T, M, S}
+    tree::T
+    mesh::M
+    strategy::S
+end
+
+function PointFinder(
```

Comment:
can you add documentation with just a example. Maybe something like:
```julia
"""
A `PointerFinder` helps locating some arbitrary coordinates on a `Mesh`. Given some coordinates `x`, the cell containing this point can be identified, and a corresponding `CellPoint` can be built to evaluate `Bcube` lazy operators.

# Example
```julia
mesh = rectangle_mesh(10,10)
pt = PointFinder(mesh)
x = rand(2)

@show icell = find_cell_index(pt, x) # find the index of the cell index containing x
@show cpoint = find_cell_point(pt, x) # build a `CellPoint` corresponding to x

u = FEFunction(TrialFESpace(FunctionSpace(:Lagrange, 1), mesh))
@show interpolate(pt, x, u) # interpolate u on x
```
"""
```